clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -413,9 +413,16 @@ DPCT_NON_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::opt<std::string>), AnalysisS
 DPCT_NON_ENUM_OPTION(
     DPCT_OPT_TYPE(static llvm::cl::opt<std::string>), ChangeExtension,
     "change-filename-extension",
-    llvm::cl::desc("Specifies which extensions of migrated files will be changed, "
-                   "split by comma. Default:\n"
+    llvm::cl::desc("DEPRECATED: Specifies which extensions of migrated files"
+                   "will be changed, split by comma. Default:\n"
                    ".cu, .cuh, .cpp, .cxx, .cc, .c, .C"),
     llvm::cl::value_desc("file"), llvm::cl::cat(DPCTCat), llvm::cl::Optional)
 
+DPCT_NON_ENUM_OPTION(
+    DPCT_OPT_TYPE(static llvm::cl::opt<bool>), LimitChangeExtension,
+    "change-cuda-files-extension-only",
+    llvm::cl::desc(
+        "Limit extension change to .cu and .cuh files only. Default: false."),
```

Comment:
`Default: false` should be `Default: off` for boolean option `--change-cuda-files-extension-only`

other similar examples:
`--report-only                           - Only reports are generated. No SYCL code is generated. Default: off.`
`--suppress-warnings-all                 - Suppresses all migration warnings. Default: off.`
`--sycl-named-lambda                     - Generates kernels with the kernel name. Default: off.`